cpp/src/fil/infer.cu

```diff
@@ -533,7 +538,7 @@ __global__ void infer_k(storage_type forest, predict_params params) {
         acc.accumulate(infer_one_tree<NITEMS, leaf_output_t<leaf_algo>::T>(
                          forest[j], cols_in_shmem ? sdata : block_input,
                          num_cols, num_input_rows),
-                       j);
+                       j, num_input_rows);
```

Comment:
fixing a bug which somehow did not trigger the test in the previous PR. It triggers it now.